api/api/routers/mcp/_mcp_models.py

```diff
@@ -613,15 +636,16 @@ def from_domain(cls, error: ErrorResponse.Error):
 
 
 class AgentVersion(BaseModel):
-    id: str
-    model: str
-    temperature: float | None
-    messages: list[Message] | None
-    instructions: str | None
-    top_p: float | None
-    max_tokens: int | None
-    frequency_penalty: float | None
-    presence_penalty: float | None
+    id: str = Field(description="The version id of the agent")
+    model: str = Field(description="The model (id) used by the version")
+    temperature: float | None = Field(description="The temperature setting of the version")
+    messages: list[Message] | None = Field(description="The messages that are part of the version")
```

Comment:
> "The messages that are part of the version"

I don't think that's clear enough. Version messages are an "odd" concept. Maybe something like:

> The message that will start every completion request message list for the version. Text content can be a Jinja2 template, in which case the template will be rendered using input variables.